code-reviews/example2/math_operations.py

```diff
@@ -0,0 +1,16 @@
+# Common mathemmatical operations
+
+def add(a, b):
+    return a + b
+
+def subtract(a, b):
+    return a - b
+
+def multiply(a, b):
+    return a * b
+
+
+def divide(a, b):
+    if a == 0:
```

Comment:
try doing the same in all 4 cases

Reply:
I'm not sure, I think this is just a special case for division. Does that make sense to you?